CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🐞 Fixed
 - Fix not possible to resend messages with failed attachments [#2966](https://github.com/GetStream/stream-chat-swift/pull/2966)
 - Fix not mentioning users if they are not fetched in the local device [#2967](https://github.com/GetStream/stream-chat-swift/pull/2967)
+- Fix `startObserver()` crash in login/logout flow [#2972](https://github.com/GetStream/stream-chat-swift/pull/2972)
```

Comment:
Can we reword this, describing it's an edge case? Reading it like this, customers will get the impression this happens frequently, which is not the case.

Reply:
Done ✅